setup.py

```diff
@@ -198,8 +198,9 @@ def get_sphinx_theme_version() -> str:
     'watchtower~=0.7.3',
 ]
 apache_beam = [
-    'apache-beam>=2.20.0',
+    'apache-beam[gcp]',
```

Comment:
I think this is the problem. This was a change implemented recently for PIP 21 compatibility and I think you should revert the line top what it was before:
```suggestion
    'apache-beam>=2.20.0',
```

Reply:
I think you simply carried the older version here and during rebase the change was reverted by mistake.

Reply:
Oh no, so sorry about this! I did have one really tricky rebase with lots of conflicts for some reason... I must have introduced this there, should have been more careful! Hopefully all will be well now